src/App/App.tsx

```diff
@@ -37,6 +37,9 @@ const withI18n = (WrappedComponent: any) => {
       }
       // set language based on URL, default to English
       const lang = locale || 'en';
+      if (lang !== 'en') {
+        registerLang('en', translations.en);
```

Comment:
so english will always be registered?
I think it would be clearer if you just wrote:
registerLang('en', translations.en);
if (lang !== 'en') {
  registerLang(lang, transaltions[lang]);
}